lineage/figures/common.py

```diff
@@ -633,4 +648,5 @@ def sort_lins(lapt_tHMMobj_list):
         else:
             st6.append(lins)
 
-    return st1[0:min(10, len(st1))] + st2[0:min(10, len(st2))] + st3[0:min(10, len(st3))] + st4[0:min(10, len(st4))] + st5[0:min(10, len(st5))] + st6[0:min(10, len(st6))]
+    return st1 + st2 + st3 + st4 + st5 + st6
+    # return st1[0:min(10, len(st1))] + st2[0:min(10, len(st2))] + st3[0:min(10, len(st3))] + st4[0:min(10, len(st4))] + st5[0:min(10, len(st5))] + st6[0:min(10, len(st6))]
```

Comment:
Do you need this commented out code? Why did this change?

Reply:
The previous data was too large to plot in one figure, so I had chose the first 10 lineages that start with a specific phase.
This MCF10A data is not that large and we can plot all of it, so I didn't select anything specific and plot all of the lineages.
I kept that line as a comment to maybe use it later should we be given a very large set of data.

Reply:
Same question.

Reply:
I will write a better function for this in a minute.

Reply:
I wrote it in a more general way now.